internal/manifests/queryfrontend/query_frontend.go

```diff
@@ -239,7 +239,6 @@ func services(tempo v1alpha1.Microservices) []*corev1.Service {
 			Labels:    labels,
 		},
 		Spec: corev1.ServiceSpec{
-			ClusterIP: "None",
```

Comment:
This line should not be deleted, this setting is required for the tempo cluster.

Reply:
Good catch! I don't know why I removed it. Sorry